### DISCUSSION TITLE
problem with fonts

### DISCUSSION THREAD

**Discussion by MayoioI** [community/community#135032] [Programming Help]
### Body

I want to change the font in my css file by using "font-family"and its not working.
.word{
font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
}

 But when i write font-family in my html file, all works
</head>
<body>
    <div class="word" style="font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;"> asdasd</div>
</body>
</html>

Can somebody explain why this not working in my css file ?

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by eliancheng** [comment]
Hello, there
I have some experience in this field.
Please use font files.

ex]
`@font-face {
  font-family: 'font name';
  src: url('path/to/font-file.woff2') format('woff2');

.word {
  font-family: 'font name', sans-serif;
}
`

This is not working, please contact me.
Thanks

**Reply by MayoioI** [reply]
i tried with windows font and download the font from internet, but its not working. Mb this is problem with my vs code

**Comment by eliancheng** [comment]
Hello
Thanks for your response.
If this is not working, please use "!important"
```
.word {
  font-family: 'Lucida Sans', sans-serif !important;
}
```

**Comment by UjjwalSaini07** [comment]
**_1. CSS File Not Linked Correctly_**
Ensure that your CSS file is properly linked to your HTML file. The link tag should be inside the <head> section of your HTML file:
```
<link rel="stylesheet" href="styles.css">
```

**_2. Specificity and Overriding Styles_**
Check if there are other CSS rules that might be overriding your .word class. CSS rules with higher specificity or inline styles might be affecting the output. You can use browser developer tools to inspect the element and see which styles are applied.

**_3. Class Not Applied Correctly_**
Ensure that the class word is applied correctly in your HTML elements:
`<p class="word">This is a sample text.</p>`

**_4. CSS File Format_**
Ensure your CSS file is saved with the correct .css extension and is in a valid CSS format.

**_5. Font Availability_**
Ensure that the fonts you are specifying are available and properly loaded. 

**Take Help from this example:**
```
.word {
  font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande', 'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
}
```

If u find this helpful so please make this as correct as answer it really very helpful for mine.
and if any problem further so let me know.

**Comment by hojjatfazayeli** [comment]
Hi @MayoioI
I studied your problem in fonts config on css file. if you want set a font in css file , please download ttf or woff format of font file then put font file on a specific directory in project file.
finally you should add font face on css file and add font url at font face config and next go to use with font-family in css file.
**for example :**
`@font-face {
    font-family: "IBMPlexMono-Bold";
    src: url("../fonts/IBMPlexMono-Bold.ttf") format("truetype");
}`

Good Luck
